src/meta/expr.lean

```diff
@@ -514,11 +514,79 @@ meta def to_binder : expr → binder
 | _                       := default binder
 
 /-- Strip-away the context-dependent unique id for the given local const and return: its friendly
-`name`, its `binder_info`, and its `type : expr`.-/
+`name`, its `binder_info`, and its `type : expr`. -/
 meta def get_local_const_kind : expr → name × binder_info × expr
 | (expr.local_const _ n bi e) := (n, bi, e)
 | _ := (name.anonymous, binder_info.default, expr.const name.anonymous [])
 
+/-- Returns an `expr.local_const`'s `binder_info`. -/
+meta def local_binder_info {elab : bool} : expr elab → binder_info
```

Comment:
This is just `expr.local_binding_info` generalized to `pexpr`, right? And I don't see it used anywhere in this PR? I'd rather generalize that in core than introduce a confusingly similar name for almost the same function.

Reply:
You're right on both counts: I missed `expr.local_binding_info` and kept this here after I stopped using it because I thought something like it didn't exist.